src/axolotl/utils/schemas/config.py

```
@@ -1253,8 +1264,21 @@ def default_dataloader_opts(cls, data):
+        if data.get("dataset_processes") is not None:
+            if data.get("dataset_num_proc") is None:
+                data["dataset_num_proc"] = data["dataset_processes"]
+                LOG.warning(
+                    "dataset_processes is deprecated and will be removed in a future version. "
+                    "Please use dataset_num_proc instead."
+                )
+            else:
+                raise ValueError(
+                    "Both dataset_processes and dataset_num_proc are set. "
+                    "Using dataset_num_proc and ignoring dataset_processes."
+                )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)